aptos-move/framework/aptos-framework/sources/aggregator/aggregator.spec.move

```diff
@@ -6,20 +6,24 @@ spec aptos_framework::aggregator {
     spec add {
         // TODO: temporary mockup.
         pragma opaque;
+        aborts_if false;
```

Comment:
This native function can actually abort. Please refer to the comment in `aggregator.move`. (cc: @georgemitenkov )

Reply:
> Remove aborts if false does not affect the validation.
I didn't get this. Could you elaborate what you mean?

According the comment in `aggregator.move`, this function "Aborts on overflowing the limit."